Source/Libraries/GSF.PhasorProtocols/UI/DataModels/RealTimeStream.cs

```diff
@@ -154,14 +157,62 @@ public string StatusColor
         /// </summary>
         public bool ConfigurationOutOfSync
         {
-            get => m_configurationOutOfSync;
+            get => m_configurationOutOfSync ?? false;
             set
             {
                 m_configurationOutOfSync = value;
                 OnPropertyChanged(nameof(ConfigurationOutOfSync));
+                OnPropertyChanged(nameof(ConfigurationOutOfSyncContent));
+                OnPropertyChanged(nameof(ConfigurationOutOfSyncToolTip));
+                OnPropertyChanged(nameof(ConfigurationOutOfSyncColor));
             }
         }
 
+        /// <summary>
+        /// Gets content message for input stream configuration sync state.
+        /// </summary>
+        public string ConfigurationOutOfSyncContent
+        {
+            get => DeviceStateUnavailable ? "Loading..." :
+                m_configurationOutOfSync is null ? "Checking Config..." :
+                    m_configurationOutOfSync.Value ?
+                        "Config Out of Sync" :
+                        "Config In Sync";
+            set { }
```

Comment:
I don't get it. Why does it have a setter if the setter isn't used and doesn't do anything?

Reply:
WPF thing - it complained when it didn't exist on binding? Maybe I can do something in WPF to make it work with a readonly property - but this worked - and this was how other properties were doing it...

Reply:
Hm, I do see it in `RealTimeDevice`, but all other read-only bindings throughout the codebase just do `Target={Binding Prop}` instead of `Target={tsfBinding:Column Path=Prop}`.

Reply:
Honestly, I am not even sure what the difference is between those two targets except that we must have "modified" one to be custom. WPF and I don't get along well. This update too way many more hours than it should have fighting little things. That said, happy to try a different way if think it will make a difference. My thought is that we can eventually abandon this code base - that `RealTimeStream` class is way too complex. Maybe it needs to be, but it so, its too much jammed into that one file...

Reply:
`tsfBinding:Column` just sets a bunch of properties on the `Binding` class. The most relevant one is probably `Mode=TwoWay`.

https://github.com/GridProtectionAlliance/gsf/blob/cd87678635087a8c15ad9aa4f791cb9478e9c58d/Source/Libraries/GSF.TimeSeries/UI/Binding/Column.cs#L33-L52

Reply:
Yeah, guess that would be why `set` is required.

Reply:
And, by extension, why it's the wrong type of binding for read-only properties. But I suppose I will drop it since I don't really care enough to fix it myself either.